src/main/java/entities/Comment.java

```diff
@@ -1,36 +1,26 @@
 package entities;
 
-
-import java.util.ArrayList;
 import java.util.Date;
-import java.util.List;
 
 /**
  * Comment extends abstract class Postable and inherits all of its
  * instance attributes (user, body, replies) and addComment method.
  *
  */
 public class Comment extends Postable{
-    // number of Comments instantiated
-    public static int numComments = 0;
-    // id is the unique identifier of Comment, equal to number of Comments at time of instantiation
-    private final int id;
 
     /**
      * Initializes an instance of Comment.
      *
      * @param commenter The user that posted this comment.
      * @param body The text content of the comment.
-     * @param numCommentsCreated The number of comments created in the program so far.
+     * @param id The unique id for this comment
      */
-    public Comment(User commenter, String body, int numCommentsCreated){
```

Comment:
In all the constructors for all the entities, we decided to only use the IDs of the other entities so instead of User commenter we'll have int commenterID. Could you please change that?